code/src/index.css

```diff
@@ -1,13 +1,235 @@
+* {
+  overflow-x: hidden;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #000;
+  color: #fff;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.root {
+  width: 100vw;
+  margin: 0;
+}
+
+header {
+  border-bottom: 0.5px solid #a0a0a0;
+  margin: 0 auto 15px auto;
+  width: 90vw;
+}
+
+h1 {
+  margin-bottom: 5px;
+  margin-top: 20px;
+}
+
+.album-wrapper{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: auto;
+  width: 100vw;
+}
+
+.album {
+  width: 90%;
+  margin: 0 10px;
+}
+
+.album-cover {
+  position: relative;
+  width: 100%;
+}
+
+.album-cover img {
+  width: 100%;
+}
+
+.overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.6);
+  opacity: 0;
+  transition: .5s ease;
+}
+
+.heart-icon {
+  position: absolute;
+  fill: #fff;
+  width: 10%;
+  top: 42%;
+  left: 14%;
+  opacity: 0;
+  transition: .5s ease;
+}
+
+.play-icon {
```

Comment:
You could also add `cursor: pointer` property. By UX standards, all clickable elements should have this cursor set